demo/admin/src/common/blocks/StandaloneMediaBlock.tsx

```diff
@@ -20,6 +20,12 @@ export const StandaloneMediaBlock = createCompositeBlock(
                 }),
             },
         },
+        tags: [
+            defineMessage({ id: "standaloneMedia.tag.image", defaultMessage: "Image" }),
+            defineMessage({ id: "standaloneMedia.tag.video", defaultMessage: "Video" }),
+            defineMessage({ id: "standaloneMedia.tag.youtube", defaultMessage: "Youtube" }),
+            defineMessage({ id: "standaloneMedia.tag.vimeo", defaultMessage: "Vimeo" }),
```

Comment:
No need to translate brands:

```suggestion
            "YouTube",
            "Vimeo",
```